code/modules/admin/verbs/adminpm.dm

```diff
@@ -159,12 +159,14 @@
 				admin_ticket_log(src, interaction_message)
 				if(recipient != src)	//reeee
 					admin_ticket_log(recipient, interaction_message)
-
+				SSblackbox.LogAhelp(current_ticket.id, "Reply", msg, src.ckey)

```

Comment:
Move all of these calls outside of the entire if block, `msg` isn't manipulated after L142 so no need for duplicate calls. This can go at the end of the proc since there is one return case.